src/components/list/template.njk

```diff
@@ -1,13 +1,14 @@
-{% if params.isBullet %}
+{% set type = params.type | lower %}
+{% if type == 'bullet' %}
```

Comment:
Would this be better as a switch statement? Could also inline the call to `lower` then.

Reply:
if only nunjucks had a switch statement...

Reply:
Dangit, nunjucks!